index.php

```diff
@@ -94,14 +94,14 @@
 				</div>
 			</div>
 		</div>
-	</div>
-	<div class="column">
-		<img src="imgs/chickenWings.jpg" />
+		<div class="column">
+			<img src="imgs/chickenWings.jpg" />
+		</div>
 	</div>
```

Comment:
not really related but not worth another issue, looks like incomplete set of tags or something. maybe missing `</div>`?

Reply:
Removed the extra div in commit 721d614b9d67b05f9c4d20affa7dabc6267861a4